source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -252,6 +252,22 @@ def enumerate(podList = nil)
               telemetryProperties["DATA_COLLECTION_INTERVAL_MINUTES"] = @run_interval / 60
             end
           end
+          begin
+            if Dir.exist?("/etc/mdsd.d/config-cache/configchunks")
+              Dir.glob("/etc/mdsd.d/config-cache/configchunks/*.json") { |file|
+                if File.file?(file) && File.exist?(file)
+                  if File.foreach(file).grep(/LINUX_SYSLOGS_BLOB/).any?
+                    telemetryProperties["syslogEnabled"] = "true"
+                  end
+                  if File.foreach(file).grep(/ContainerInsightsExtension/).any? && File.foreach(file).grep(/dataCollectionSettings/).any?
+                    telemetryProperties["dataCollectionSettingsEnabled"] = "true"
```

Comment:
we can remove dataCollectionSettingsEnabled since this already taken care at line#244 - https://github.com/microsoft/Docker-Provider/blob/4c1ca9eff6e928c470e224e48be76e11c31f4774/source/plugins/ruby/in_kube_podinventory.rb#L244